src/pages/AdminPages/UnapprovedAccounts/index.js

```diff
@@ -0,0 +1,163 @@
+import React, {useEffect, useState} from 'react'
```

Comment:
This component is doing a lot. Let's trim this down.

Make each unapproved user row a component...that way the logic won't be so complex. And try utilizing the Form Components built. I've used it a couple of times so you can get an idea of how it works.

Reply:
Thanks, I'd do this now